src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheckTest.java

```diff
@@ -146,6 +146,14 @@ public void test1322879() throws Exception {
                expected);
     }
 
+    @Test
+    public void testTrimFunc() throws Exception {
```

Comment:
Please change `testTrimFunc` to `testTrimOptionProperty` and do something similar to the file name.

I wouldn't worry about shortening things like "Func" instead of "Function". It is better to express what is being done as long as it is within a reasonable amount of text, then for it to be vague and lose meaning over time and personnel.

Reply:
done